src/main/java/org/example/Main.java

```diff
@@ -1,7 +1,43 @@
 package org.example;
 
+import java.util.Scanner;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Enter the first number:");
+        double num1 = scanner.nextDouble();
+
+        System.out.println("Enter the second number:");
+        double num2 = scanner.nextDouble();
+
+        System.out.println("Select the operation (+, -, x, /):");
+        String operation = scanner.next();
+
+        double result = calculate(num1, num2, operation);
+        System.out.println("Result: " + result);
+    }
+
+    private static double calculate(double num1, double num2, String operation) {
+        switch (operation) {
+            case "+":
+                return num1 + num2;
+            case "-":
+                return num1 - num2;
+            case "x":
+                return num1 * num2;
+            case "/":
+                if (num2 != 0) {
+                    return num1 / num2;
+                } else {
+                    System.out.println("It is not possible to divide by zero.");
```

Comment:
It's better to throw an exception or at least use System.err.println() instead of sout